extensions/amp-story-interactive/0.1/test/validator-amp-story-interactive-valid.html

```diff
@@ -76,6 +76,43 @@
       </amp-story-grid-layer>
     </amp-story-page>
     <amp-story-page id="4">
+      <amp-story-grid-layer template="fill">
+        <!-- Valid: Example of amp-story-interactive-img-poll -->
+        <amp-story-interactive-img-poll
```

Comment:
Can you add `option-x-results-category` attributes here so we verify that they work as expected?